libtsuba/include/tsuba/RDG.h

```diff
@@ -26,6 +26,16 @@ class RDGMeta;
 class RDGCore;
 struct PropStorageInfo;
 
+struct KATANA_EXPORT RDGLoadOptions {
+  /// nullopt means the partition associated with the current host's ID will be
+  /// loaded
+  std::optional<uint32_t> partition_to_load;
+  /// nullptr means all node properties will be loaded
+  const std::vector<std::string>* node_props{nullptr};
+  /// nullptr means all edge properties will be loaded
+  const std::vector<std::string>* edge_props{nullptr};
```

Comment:
Use `node_properties` and `edge_properties`. We are trying to avoid contractions in the public API when they are not required or universally used.

Don't just document the empty case. Also document the full case. It's not obvious from context what the list of properties is used for.